#!/usr/bin/env python
PACKAGE = "omav_local_planner"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("a_max_ang", double_t, 0, "Max ang acc", 4, 0, 20)
gen.add("a_max_trans", double_t, 0, "Max lin acc", 1, 0, 20)
gen.add("v_max_ang", double_t, 0, "Max ang vel", 4, 0, 20)
gen.add("v_max_trans", double_t, 0, "Max lin vel", 1, 0, 20)
gen.add("takeoff_z", double_t, 0, "Take off height over current position", 1, 0, 3)

exit(gen.generate(PACKAGE, "Config", "VoliroTrajectories"))
